docs/contribution-guidelines.md

```diff
@@ -28,72 +28,86 @@ Streamability V2 is attempting to have a linear commit history on `main` as well
 Creating and pushing a new feature branch is quite simple. Follow the steps below:
 
 1. Checkout to `develop` and pull the most recent changes.
-```s
+
+```sh
 git pull origin develop
 ```
 
 2. Create a new feature branch using the proper naming convention.
-```s
-git checkout -b <"issue-num"-"issue-type"-"branch-name">
+
+```sH
+git checkout -b <"issue-num"-"branch-name">
 ```
-> NOTE: If you do not have an issue number or type, `NA-0`, should be used instead followed by the branch name.
+> NOTE: If you do not have an issue number or type, `0`, should be used instead followed by the branch name.
 > 
 > The branch name is whatever the developer thinks fits best but should be descriptive, lowercase, and hyphen separated. 
 
 3. Add your changes to the branch and create a new commit. The commit should contain a descriptive message of the changes or additions you have made. It is also a good idea to run an es-lint check first.
-```s
+
+```sh
 npm run lint
```

Comment:
This uses `npm run lint` in the example, perhaps the `# Code Quality` section should come first?